webapp/src/components/__tests__/ProgressBar.test.tsx

```diff
@@ -0,0 +1,10 @@
+import { render, screen } from '@testing-library/react';
+import ProgressBar from '../ProgressBar';
+import React from 'react';
+
+describe('testing if progress bar inside Profile component is rendering', () => {
+  it('should render progress bar', () => {
+    render(<ProgressBar />);
+    expect(screen.getByText('Learning Progress')).toBeInTheDocument();
+  });
```

Comment:
Do we need to catch errors here or not?

Reply:
We wanted to test whether the ProgressBar component would render or not.
I don't think we need to catch errors here.
I hope I answered your question Nazanin.